### PR TITLE
Cypher syntax change

### PR DESCRIPTION
**Pull request by martin-neotech**
`CREATE CONSTRAINT ON ... ASSERT ...`

Changed in `4.4` to

`CREATE CONSTRAINT FOR ... REQUIRE ...`

### CODE REVIEW

File: modules/ROOT/pages/cypher-intro/schema.adoc [hunk 1]
```diff
@@ -100,19 +100,29 @@ For example: "If a node has a label of `Actor` and a property of `name`, then th
...
 Adding the unique constraint will implicitly add an index on that property.
 If the constraint is dropped, but the index is still needed, the index will have to be created explicitly.
 
 Constraints can be added to database that already has data in it.
 This requires that the existing data complies with the constraint that is being added.
 
-We can inspect our database to find out what constraints are defined.
-We do this by calling the built-in procedure `db.constraints`:
+You can inspect a database to find out what constraints are defined.
+Check the defined constraints by calling the built-in procedure `db.constraints`:
```
Comment (Hunterness) [inline modules/ROOT/pages/cypher-intro/schema.adoc:125]: shouldn't this be the `SHOW CONSTRAINTS` command?

Comment (Hunterness) [reply modules/ROOT/pages/cypher-intro/schema.adoc:125]: also the result from the procedure might have been updated as well, but I'm not 100% sure, @Lojjs did this particular syntax update I believe so she might have better answer

We definitely have different result from the command

Comment (martin-neotech) [reply modules/ROOT/pages/cypher-intro/schema.adoc:125]: > shouldn't this be the `SHOW CONSTRAINTS` command?

Yes sure if there is a new `SHOW CONSTRAINTS` command in 4.4 :)

Comment (Hunterness) [reply modules/ROOT/pages/cypher-intro/schema.adoc:125]: I know when the command was introduced, I was just surprised it was not used here

**Review by Lojjs — APPROVED** [review]
Looks correct to me (Therese asked me to have a look)

File: modules/ROOT/pages/cypher-intro/load-csv.adoc [hunk 2]
```diff
@@ -233,14 +233,14 @@ Using _Neo4j Browser_, run the following Cypher:
 
 [source, cypher, indent=0]
 ----
-CREATE CONSTRAINT personIdConstraint ON (person:Person) ASSERT person.id IS UNIQUE
+CREATE CONSTRAINT personIdConstraint FOR (person:Person) REQUIRE person.id IS UNIQUE
 ----
 
 Or using xref:4.4-preview@operations-manual:ROOT:tools/cypher-shell/index.adoc#cypher-shell[_Neo4j Cypher Shell_], run the command:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/cypher-intro/load-csv.adoc:239]: This format is different from how we link between manuals elsewhere?

**Review by AlexicaWright — COMMENTED** [review]
LGTM, just one question on the format for links between manuals.

File: modules/ROOT/pages/cypher-intro/schema.adoc [hunk 3]
```diff
@@ -100,34 +100,45 @@ For example: "If a node has a label of `Actor` and a property of `name`, then th
...
 
-+------------------------------------------------------------+
-| description                                                |
-+------------------------------------------------------------+
-| 'CONSTRAINT ON (movie:Movie) ASSERT movie.title IS UNIQUE' |
-+------------------------------------------------------------+
++----------------------------------------------------------------------------------------------------+
+| id | name                  | type         | entityType | labelsOrTypes | properties | ownedIndexId |
++----------------------------------------------------------------------------------------------------+
+| 4  | "constraint_751eb8ab" | "UNIQUENESS" | "NODE"     | ["Movie"]     | ["title"]  | 3            |
```
Comment (Hunterness) [inline modules/ROOT/pages/cypher-intro/schema.adoc:140]: I can add that there is more columns when using `YIELD`, including the create command for the constraint which is more similar to what was here before

```
SHOW CONSTRAINTS YIELD createStatement
```

**Pull request merged**